tests/conftest.py

```diff
@@ -0,0 +1,65 @@
+import logging
```

Comment:
I think this will affect all tests under /tests directory.
Please create a new dir with your test.py and conftest.py

Reply:
Why? I understand that it's not fully reusable fixture (mainly because nobody else uses the fixtures properly yet), but in current state, I would rather keep it here.